src/pages/FarmPage/FarmsList.tsx

```diff
@@ -487,37 +456,33 @@ const FarmsList: React.FC<FarmsListProps> = ({ bulkPairs, farmIndex }) => {
           ))}
         </Box>
       )}
-      {farmIndex === GlobalConst.farmIndex.LPFARM_INDEX &&
-      stakingInfos &&
-      !pageloading ? (
-        stakingInfos.map((info: StakingInfo, index) => (
-          <FarmLPCard
-            key={index}
-            dQuicktoQuick={Number(lairInfo.dQUICKtoQUICK.toSignificant())}
-            stakingInfo={info}
-            stakingAPY={stakingAPYs[index]}
-          />
-        ))
-      ) : farmIndex === GlobalConst.farmIndex.DUALFARM_INDEX &&
-        stakingDualInfos &&
-        !pageloading ? (
-        stakingDualInfos.map((info: DualStakingInfo, index) => (
-          <FarmDualCard
-            key={index}
-            dQuicktoQuick={Number(lairInfo.dQUICKtoQUICK.toSignificant())}
-            stakingInfo={info}
-            stakingAPY={stakingAPYs[index]}
-          />
-        ))
-      ) : (
-        <>
-          <Skeleton width='100%' height={100} />
-          <Skeleton width='100%' height={100} />
-          <Skeleton width='100%' height={100} />
-          <Skeleton width='100%' height={100} />
-          <Skeleton width='100%' height={100} />
-        </>
-      )}
+      {farmIndex === GlobalConst.farmIndex.LPFARM_INDEX && stakingInfos
```

Comment:
Ideally we should. just check for  this `(!stakingInfos || !stakingDualInfos)`  first, so we don't need expand out the conditions for the ternary

Reply:
Remove ternary conditions, Add declarative conditions for each piece of UI content

`{(farmIndex === GlobalConst.farmIndex.LPFARM_INDEX && !stakingInfos) ||
        (farmIndex === GlobalConst.farmIndex.DUALFARM_INDEX &&
          !stakingDualInfos && (
            <>
              <Skeleton width='100%' height={100} />
              <Skeleton width='100%' height={100} />
              <Skeleton width='100%' height={100} />
              <Skeleton width='100%' height={100} />
              <Skeleton width='100%' height={100} />
            </>
          ))}
      {farmIndex === GlobalConst.farmIndex.LPFARM_INDEX &&
        stakingInfos &&
        stakingInfos.map((info: StakingInfo, index) => (
          <FarmLPCard
            key={index}
            dQuicktoQuick={Number(lairInfo.dQUICKtoQUICK.toSignificant())}
            stakingInfo={info}
            stakingAPY={getPoolApy(info?.pair)}
          />
        ))}
      {farmIndex === GlobalConst.farmIndex.DUALFARM_INDEX &&
        stakingDualInfos &&
        stakingDualInfos.map((info: DualStakingInfo, index) => (
          <FarmDualCard
            key={index}
            dQuicktoQuick={Number(lairInfo.dQUICKtoQUICK.toSignificant())}
            stakingInfo={info}
            stakingAPY={getPoolApy(info?.pair)}
          />
        ))}`